### ISSUE TITLE
Absorption coefficients in tissue generator not correct for blood vessels

### ISSUE THREAD

**Issue by mschllnbrg** [CAMI-DKFZ/simpa#40]
correctly assign absorption coefficients in blood vessels: 

set water volume fraction to 0 in the tissue_library for the following classes: 

- blood_generic 
- blood_venous
- blood_arterial

**Comment by jgroehl** [comment]
I propose we add both automatic and visual test cases to test the molecular tissue compositions that are automatically provided by the TissueLibrary class.

For custom molecular compositions, we cannot really provide such an automatic test, but we can provide both (1) utility functions to allow for easy visualisation of the created composition and (2) an automatic generic test that tests randomly created tissue compositions and their mixing behaviour.

- [ ] Write a test that automatically reproduces and identifies the described bug
- [ ] Write a generic test for molecular compositions and tissues
- [ ] Create utility methods that allow for easy visualisation of the ranges that a certain tissue type or molecular composition will achieve
- [ ] Create utility methods that allow for easy visualisation of the ranges that are present in a simulated volume or molecular composition will achieve

**Comment by jgroehl** [comment]
added new label [CRITICAL](https://github.com/CAMI-DKFZ/simpa/labels/CRITICAL) to reflect the urgency of this bug

**Comment by jgroehl** [comment]
Added branch `T40-TestsForTissueLibrary` with a first simple test (volume fractions <= 1) that already fails, even after hotfix.

**Comment by jgroehl** [comment]
We now have tests, that automatically test if the tissues that are provided by our `TissueLibrary` are within 10% of the optical and acoustic property values that we have reviewed from literature in #41.

Remaining issues:
For some tissue types we did not find literature references and are using best guess values. (Maybe make the literature review thing an ongoing issue?)
The power-law fit for scattering from Mie Theory, that we assumed for the scattering properties of our molecules usually does not seem to hold for wavelengths smaller than approx. 650 nanometers.
Currently, all scattering spectra are still assumed follow the theory. nevertheless, with the inclusion of #56, we can now also include different spectra and might want to do so in the future.
